ckanext/budgetviz/groupbarchart_munc/theme/public/custom.css

```diff
@@ -15,6 +15,9 @@
 }
 
 .chartarea {
-    margin-top: 20px;
+    padding: 10px 25px;
```

Comment:
I read below mobile and tab not supported, just out of curiosity how many resolutions have we tested it out so far ?